### ISSUE TITLE
remove cJSON and tinyxml2 libraries

### ISSUE THREAD

**Issue by balazsracz** [bakerstu/openmrn#72]
these are not used at all.

**Comment by bakerstu** [comment]
I'm in favor of getting rid of them (unless JohnSL needs C JSON).  Then again, including these as part of the OpenMRN package goes against the grain of providing dependent packages separately so that we are not redistributing someone else copyrighted material.

**Comment by balazsracz** [comment]
@JohnSL do you need that specific JSON library? Or would you prefer to pick your own, or upgrade and move to the same distribution policy as all other libraries (i.e. external in /opt).

**Comment by JohnSL** [comment]
I agree with @bakerstu that they shouldn't be in OpenMRN. I will be using JSON for my own application, and can include the library there.
